routes.go

```diff
@@ -309,6 +309,7 @@ func (h *Headscale) handlePrimarySubnetFailover() error {
 		}
 	}
 
+	h.setLastStateChangeToNow()
```

Comment:
This should only run if subnets were actually failed over.

Reply:
Should be fixed.

Reply:
Thanks!